changedetectionio/tests/unit/test_notification_diff.py

```diff
@@ -16,9 +16,9 @@ def test_expected_diff_output(self):
         output = diff.render_diff(previous_file=base_dir + "/test-content/before.txt", newest_file=base_dir + "/test-content/after.txt")
         output = output.split("\n")
         self.assertIn('(changed) ok', output)
-        self.assertIn('(into   ) xok', output)
-        self.assertIn('(into   ) next-x-ok', output)
-        self.assertIn('(added  ) and something new', output)
+        self.assertIn('(into) xok', output)
+        self.assertIn('(into) next-x-ok', output)
+        self.assertIn('(added) and something new', output)
```

Comment:
can you add also an assert for `diff_removed` ?

Reply:
added